notary-signer/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.8
+FROM alpine:3.10
 
 ENV TAG v0.6.1
```

Comment:
I think the tag will need to be updated? to v0.6.1-2? Not sure how we do these

Reply:
This refers to the tags in the notary repo; https://github.com/docker/notary-official-images/blob/3ef28cbe319cf001a46641ddb12a3b767568959f/notary-server/Dockerfile#L15

And that tag didn't change; the `-2` is a packaging-only change, so this repository will need a new tag